src/site/content/en/blog/streams/index.md

```diff
@@ -341,37 +341,24 @@ for await (const chunk of stream) {
   Asynchronous iteration is not yet implemented in any browser.
 {% endAside %}
 
-A workaround to use asynchronous iteration today is to implement the behavior with a helper
-function. This allows you to use the feature in your code as shown in the snippet below.
+A workaround to use asynchronous iteration today is to implement the behavior with a polyfill.
 
 ```js
-function streamAsyncIterator(stream) {
-  // Get a lock on the stream:
-  const reader = stream.getReader();
-
-  return {
-    next() {
-      // Stream reads already resolve with {done, value}, so
-      // we can just call read:
-      return reader.read();
-    },
-    return() {
-      // Release the lock if the iterator terminates.
+if (ReadableStream.prototype[Symbol.asyncIterator]) {
```

Comment:
Shouldn't there be a leading `!` here, so that you only do the assignment if it's not already defined?

Reply:
Duh, of course. Forgot to negate after testing in the Console. Good catch.